app/src/main/java/com/cornellappdev/android/eateryblue/ui/screens/EateryDetailScreen.kt

```diff
@@ -889,9 +889,13 @@ fun EateryMenuWidget(event: Event) {
                         }
                     }
                 }
+                if (categoryIndex != event.menu!!.lastIndex) {
+                    Divider(color = GrayZero,
+                        modifier = Modifier
+                            .height(10.dp)
+                    )
```

Comment:
There's some sus spacing here; make sure to run the formatter on your code before pushing:
![image](https://github.com/cuappdev/eatery-blue-android/assets/47724806/4d7e0a51-79f7-4444-868e-c364afd70072)
Right click the code folder and hit "Reformat Code".

Reply:
fixed! (first photo is before, second is after)
<img width="245" alt="Screenshot 2024-02-28 at 2 46 04 PM" src="https://github.com/cuappdev/eatery-blue-android/assets/149902487/073af176-aa1d-48ca-81b2-7ff568082d61">
<img width="277" alt="Screenshot 2024-02-28 at 5 28 17 PM" src="https://github.com/cuappdev/eatery-blue-android/assets/149902487/60a497d3-3b47-4baf-8a8d-812efcf203cb">